bitcoin/psbt.c

```diff
@@ -20,6 +20,18 @@
 	memmove((arr) + (pos), (arr) + (pos) + 1,		\
 		sizeof(*(arr)) * ((num) - ((pos) + 1)))
 
+/* FIXME: someday this will break, because it's been exposed in libwally */
+int wally_psbt_clone(struct wally_psbt *psbt, struct wally_psbt **output)
```

Comment:
~Isn't this an exact copy of the one found in `tx.c` added in commit 08a677de5e90b01bdb55f6b8a4b0a2a2f839d44c?~

Edit: Nope, it isn't `psbt` and `tx` are different primitives and warrant their own clone function :+1: